File: patterns/javascript-patterns/pattern140.js
// Pattern 140: ...

//     *
//    * *
//   *****
//  *     *
// *       *

let n = 5;
let x = n;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= 2 * n; j++) {
        if (j === x || j === n + i - 1) {
            row += '*';
        } else if (j >= x && j <= n + i - 1 && i === Math.floor(n / 2) + 1) {
            row += '*';
        } else {
            row += ' ';
        }
    }
    console.log(row);
    x--;
}

File: patterns/python-patterns/pattern140.py
# Pattern 140: ...

#     *
#    * *
#   *****
#  *     *
# *       *

n = 5
x = n

for i in range(1, n + 1):
    for j in range(1, 2 * n + 1):
        if j == x or j == n + i - 1:
            print("*", end="")
        elif j >= x and j <= n + i - 1 and i == n // 2 + 1:
            print("*", end="")
        else:
            print(" ", end="")
    print()
    x -= 1
